test/test_foreach.py

```diff
@@ -799,33 +800,6 @@ def test_div_list(self, device, dtype):
             self.assertEqual(res, tensors1)
             self.assertEqual(tensors1, res)
 
-    def test_bin_op_list_error_cases(self, device):
-        tensors1 = []
-        tensors2 = []
-
-        for bin_op in self.foreach_bin_ops + self.foreach_bin_ops_:
-            # Empty lists
-            with self.assertRaises(RuntimeError):
-                bin_op(tensors1, tensors2)
-
-            # One empty list
-            tensors1.append(torch.tensor([1], device=device))
-            with self.assertRaises(RuntimeError):
-                bin_op(tensors1, tensors2)
-
-            # Lists have different amount of tensors
-            tensors2.append(torch.tensor([1], device=device))
```

Comment:
I don't see this test anymore - did I miss it?

Reply:
These test cases were duplicated. You can see them in `test_bin_op_list_error_cases`

Reply:
Oh thanks, I'm blind.